archinstall/default_profiles/desktops/plasma.py

```diff
@@ -16,7 +16,11 @@ def packages(self) -> list[str]:
 			"konsole",
 			"kate",
 			"dolphin",
+			"dolphin-plugins",
+			"flatpak-kcm",
+			"kcmutils",
```

Comment:
flatpak and especially `kcm` is optional packages and not required for a bare minimum experience that still feels like a decent experience I think.